.travis.yml

```diff
@@ -1,12 +1,17 @@
 language: node_js
+node_js: 8
 sudo: false
-node_js:
-- '8'
```

Comment:
Does changing this from a list to an int continue to work in Travis CI?

Reply:
Yes. Also see the CI run for this PR.